tensorflow/compiler/xla/service/hlo_parser.cc

```diff
@@ -636,7 +636,7 @@ bool HloParserImpl::ParseInstructionList(HloComputation** computation,
     // the pool, which should not happen.
     if (root_node == nullptr) {
       LOG(FATAL) << "instruction " << root_name
-                 << " was marked as ROOT but the parser has not seen it before";
+                 << " was marked as ROOT but the parser has not seen it before"; // abort()
```

Comment:
Sorry for being so pedantic, but can you please use a full english sentence here?  `// abort()` reads like code that was accidentally commented out.

Reply:
Sorry, please check now. Thanks.